src/main/java/edu/uwaterloo/ece658/entity/User.java

```diff
@@ -46,6 +48,9 @@
 
     private String gender;
 
+    @ElementCollection
+    private Map<String, String> subscriptionArns;
```

Comment:
Why having this map?

Reply:
The way the arns work. When you create a topic you get a topicArn. When you subscribe to a topic the user gets a subscriptionArn. I wanted this so i could keep track of which topicArn the users subscriptionArn mapped to